sources/engine/Xenko.Engine/Rendering/LightProbes/BowyerWatsonTetrahedralization.cs

```diff
@@ -773,13 +773,13 @@ private unsafe bool IsPointInCircumsphere(ref Vector3 p, Vector3[] points, ref T
                 return insphere(ref points[tetrahedronPointer->Vertices[0]], ref points[tetrahedronPointer->Vertices[1]], ref points[tetrahedronPointer->Vertices[2]], ref points[tetrahedronPointer->Vertices[3]], ref p) > 0.0f;
 
                 // Equivalent without Adaptive Precision Floating-Point Arithmetic
-                var matrix = new Matrix(
+                /*var matrix = new Matrix(
```

Comment:
@xen2 Should we remove that old code instead of commenting it out?

Reply:
Let's add a bool const to be able to switch to that "adaptive precision" code path (used in multiple locations)